apps/testingaccessibility/src/server/ability.ts

```diff
@@ -5,7 +5,7 @@ import {
   hasValidPurchase,
   hasInvoice,
 } from '../utils/purchase-validators'
-import {PurchaseStatus} from '../utils/purchase-status'
+import {PurchaseStatus} from '@skillrecordings/skill-api/dist/enums/purchase-status'
```

Comment:
this was a head scratcher, but basically you can't import from root because it introduces node apis that the client side chokes on